src/services/historyServices/getHistory.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+import { addToast } from "../../features/features";
+
+export const getHistory = async (_, rejectWithValue, dispatch) => {
+  try {
+    const { data } = await axios.get("/api/user/history", {
+      headers: { authorization: localStorage.getItem("token") },
+    });
+    return data;
+  } catch (error) {
+    dispatch(
+      addToast({
+        type: "WARNING",
+        desc: "Error getting History.",
+      })
+    );
+    return rejectWithValue("Error getting History.");
+  }
+};
```

Comment:
Why are we adding toast related data in this service? Shouldn't it be kept separate from the data fetch API call?

Reply:
From what I think you're asking, I want to show the toast if the error occurs. Which won't necessarily be the case usually.